src/components/pager/pager.component.js

```diff
@@ -51,36 +100,46 @@ const Pager = (props) => {
     const show = I18n.t('pager.show', { defaultValue: 'Show' });
     const elem = (
       <PagerSizeOptionsInnerStyles>
-        <span>{ show }</span>{ sizeSelector() }<span>{ records(props.pageSize) }</span>
+        <span>{ show }</span>{ sizeSelector() }<span>{ records(currentPageSize) }</span>
       </PagerSizeOptionsInnerStyles>
     );
 
-    return props.showPageSizeSelection ? elem : null;
+    return showPageSizeSelection ? elem : null;
   }
 
   return (
     <PagerContainerStyles data-component='pager'>
       <PagerSizeOptionsStyles>{pageSizeOptions()}</PagerSizeOptionsStyles>
       <PagerNavigation
         { ...props }
-        currentPage={ currentPage }
-        setCurrentPage={ setCurrentPage }
-        setCurrentThemeName={ setCurrentTheme }
+        pageSize={ currentPageSize }
+        currentPage={ page }
+        setCurrentPage={ setPage }
+        totalRecords={ totalRecords || 0 }
+        onNext={ handleOnNext }
+        onPrevious={ handleOnPrevious }
+        onFirst={ handleOnFirst }
+        onLast={ handleOnLast }
+        onPagination={ onPagination }
       />
-      <PagerSummaryStyles>{ props.totalRecords } { records(props.totalRecords) }</PagerSummaryStyles>
+      <PagerSummaryStyles>{ totalRecords || 0 } { records(totalRecords) }</PagerSummaryStyles>
     </PagerContainerStyles>
   );
 };
 
 Pager.propTypes = {
+  onNext: PropTypes.func,
+  onFirst: PropTypes.func,
+  onPrevious: PropTypes.func,
+  onLast: PropTypes.func,
   /** Current visible page */
-  currentPage: PropTypes.string.isRequired,
+  currentPage: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
is the required option removed on purpose?

Reply:
yes, because of default is set to 1